tests/external_executor/external_executor.py

```diff
@@ -145,17 +153,29 @@ def test_executor_registration(network, args):
 def test_simple_executor(network, args):
     primary, _ = network.find_primary()
 
-    wikicacher_executor = WikiCacherExecutor(primary.get_public_rpc_address())
-    supported_endpoints = wikicacher_executor.get_supported_endpoints({"Earth"})
+    if os.environ.get("CONTAINER_NODES"):
```

Comment:
I don't think we currently run the external executor e2e test with this environment variable set anywhere do we? I feel like we could always run the wiki cacher container in its own container and leave the other test in a threaded executor for now?

Reply:
Split the tests into ones which always run containerised (currently just test_simple/wiki_cacher_executor) and tests which will eventually be containerised in following PRs https://github.com/microsoft/CCF/pull/5018/commits/83c77cbcb53b48a6634df45dd89987fe4de56663